Source/Controllers/Other/SQLiteHistoryManager.swift

```diff
@@ -344,10 +345,9 @@ typealias SASchemaBuilder = (_ db: FMDatabase, _ schemaVersion: Int) -> Void
 
     /// Handles db fails
     /// - Parameters:
-    ///   - statement: Int - the command that failed
-    ///   - db: FMDatabase - the FMDatabase instance
+	///   - error: the thrown Error
     /// - Returns: nothing, should crash
-    func failedAt(_ error: Error) {
+    func failed(_ error: Error) {
```

Comment:
personally I would remove the `_` and have just `failed(error: Error)` because you then read it (and Obj-C translates it as `failed with error: error` which sounds better I think :) but nitpick!